frontend/src/components/admin-page/table.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
```

Comment:
We will reuse the table, so it would be better to move it to the common components